inst/benchmarking.Rmd

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
Maybe this file should be in a different folder? I am not sure whether we want to include the benchmarking files in the installed package on the user's machines. Maybe a folder like `bench`